ttnn/cpp/ttnn/operations/data_movement/reshape_on_device/device/reshape_op.cpp

```diff
@@ -26,10 +26,10 @@ void ReshapeDeviceOperation::validate(const std::vector<Tensor>& input_tensors)
 
     TT_FATAL(
         input_tensor_a.memory_config().memory_layout == TensorMemoryLayout::INTERLEAVED,
-        "Reshape does not currently support sharding");
+        "Use view_unsafe for reshaping sharded inputs");
```

Comment:
Since it's in experimental we should say "Use ttnn::experimental::reshape/ttnn.experimental.reshape..."